packages/andive/src/components/box.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const BoxRoot = styled.div`
+  padding: ${props => props.theme.padding}px;
+  width: 100%;
+`
+
+export interface Props {
+  className: string;
+  children?: any;
```

Comment:
I'd use React.ReactNode for children as shown here: https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#useful-react-prop-type-examples